### PR TITLE
Combined fixes for TWW

### PR DESCRIPTION
**Pull request by nulian**
Combined the fixes for TWW of several people and fixed some things myself got it running and working options.
Though I have done no extensive testing to see if everything works.

Use what you want from it.

### CODE REVIEW

**Comment by nulian** [comment]
Others are 
https://github.com/Hyphie24/ShadowedUnitFrames

And this MR to replace deprecated unitframes

https://github.com/Nevcairiel/ShadowedUnitFrames/pull/48

**Comment by Hyphie24** [comment]
Using both yours and mine, can load and move frames, but frames get "stuck" on screen when you go to relock them (this was the issue I was getting)

did some digging & I fixed it (temporarily) Line 482 of Layout.lua

Without digging fully into why it's dividing by 0 with the error it's giving, but just commenting out that line fixes that issue for now.

File: modules/range.lua [hunk 1]
```diff
@@ -81,24 +81,38 @@ local function checkRange(self)
 	end
 end
 
+local function isUsable(key)
```
[the remark this answers is not in the document]
Comment (argium) [reply modules/range.lua:84]: There are a lot of references to `GetSpellInfo` in the options addon. If Blizzard has standardised everything to the table structure, then the correct fix would be to update all business logic to use the new structure, and then handle migrating the existing spell IDs that have been saved in the profile.

config.lua lines 499-529

**Review by argium — CHANGES_REQUESTED** [review]

**Comment by nulian** [comment]
Closing because most of these bugs are fixed in latest version

**Pull request closed without merging**